openpype/hosts/resolve/api/lib.py

```diff
@@ -179,8 +184,19 @@ def create_bin(name: str, root: object = None) -> object:
         return media_pool.GetCurrentFolder()
 
 
-def create_media_pool_item(fpath: str,
-                           root: object = None) -> object:
+def remove_media_pool_item(media_pool_item: object) -> bool:
+    media_pool = get_current_project().GetMediaPool()
+    return media_pool.DeleteClips([media_pool_item])
+
+
+def create_media_pool_item(
+        fpath: str,
+        frame_start: int,
+        frame_end: int,
+        handle_start: int,
+        handle_end: int,
+        root: object = None,
+) -> object:
```

Comment:
I have no idea why the code that is IN the targeted branch also shows up here as "new" code. I didn't change these lines for example. Any ideas why?

Reply:
Nevermind - resolved now by re-merging that branch into this one.